gradle/distributionTesting.gradle

```diff
@@ -54,16 +54,17 @@ distributionTestTasks.all { DistributionTest task ->
 
                 gradleHomeDir = rootProject.intTestImage.destinationDir
                 gradleUserHomeDir = rootProject.file('intTestHomeDir')
-                toolingApiShadedJarDir = rootProject.project(':toolingApi').shadedJar.outputDir
+                toolingApiShadedJarDir = rootProject.project(':toolingApi').testToolingApiShadedJar.destinationDir
 
                 if (requiresLibsRepo) {
                     libsRepo = rootProject.file('build/repo')
                 }
                 if (requiresDists) {
                     distsDir = rootProject.distsDir
+                    systemProperties['integTest.distZipVersion'] = version
```

Comment:
That is the timestamped version. We still have the timestamped version in the docs and the distribution tests check this. This means that we cannot cache those tests.